integration-cli/docker_api_containers_test.go

```diff
@@ -1612,3 +1614,28 @@ func (s *DockerSuite) TestPostContainersCreateWithOomScoreAdjInvalidRange(c *che
 		c.Fatalf("Expected output to contain %q, got %q", expected, string(b))
 	}
 }
+
+func (s *DockerSuite) TestContainerApiStartNoAutoCreating(c *check.C) {
+	testRequires(c, DaemonIsLinux)
```

Comment:
Not sure why Linux? Looks like this test is written to run against a Windows daemon...